ui/branding/semantic-ui/less/default/elements/button.variables

```diff
@@ -1,9 +1,7 @@
 @searchButtonColor: @green;
 
-
-
-@blackHover:none;
-@basicBorderSize :0px;
-@blackFocus :none;
-@blackActive :none;
-@blackDown:none;
\ No newline at end of file
+@blackHover: none;
```

Comment:
Do we want this here?

Reply:
@mirekys Zdar, In my opinion, the variables i.e. button.variables or checkbox.variables, is not necessary or desirable, because it unexpectedly breaks other things and it is hard to even know how it will impact the entire app. As I don't know how these variables relate to the tree select field and what they do, I did not want to break the component by removing these files. If that is OK, I propose @AlbinaMakisheva can take a look and make some refactor where we don't use the .variables files in this way, but rather keep the necessary styles for the tree select field in its .less component.

Reply:
@Ducica its already fixed in new pr https://github.com/Narodni-repozitar/nr-docs/pull/117

Reply:
@AlbinaMakisheva Great, thanks